ts/features/pn/components/F24sWithBottomSheet.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from "react";
```

Comment:
Same as `F24s.tsx` imho, I think something along the lines of `F24ListBottomSheet.tsx` would benefit the readability for someone ho does not have and/or lost context

Reply:
Done it